.chloggen/drosiek-deprecate-locked-connects.yaml

```
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: mysqlreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "deprecate `mysql.locked_accounts` in favor of `mysql.connection.errors`"
```

```suggestion
note: "deprecate `mysql.locked_connects` in favor of `mysql.connection.errors`"
```

It looks like the actual metric is `locked_connects`, not `locked_accounts`. It might be good to update the PR title to match this as well.

---

Thanks for catching that